segment_pointcloud_topic: "/rgbd_segmentation_node/segment_pointcloud"
world_frame: "world"
sensor_frame: ""

voxel_size: 0.01
voxels_per_side: 8
voxel_carving_enabled: false
allow_clear: false
anti_grazing: true
truncation_distance_factor: 3.0
use_const_weight: false
max_ray_length_m: 3

using_ground_truth_segmentation: false

object_tracking:
  enable: true
  icp_use_reciprocal_correspondences: true
  icp_max_correspondence_distance: 100
  icp_use_symmetric_objective: false
  icp_max_iterations: 100
  icp_transformation_epsilon: 0.000002
  icp_absolute_mse: 0.000000000001
  icp_euclidean_fitness_epsilon: 0.0001 # 0.01 = improvement of 1% of previous MSE.

meshing:
  update_mesh_every_n_sec: 1.0
  publish_mesh: false
  mesh_filename: "tpp_map.ply"

visualizer:
  enable: true
  camera_intrinsics: [564.3, 0, 480, 0, 564.3, 270, 0, 0, 1]
  write_frames_to_file: false
  export_path: ""

debug:
  verbose_log: false
